src/CometWrapper.sol

```diff
@@ -90,50 +90,59 @@ contract CometWrapper is ERC4626, CometHelpers {
     /// @return shares The amount of shares of the owner that are burned
     function withdraw(uint256 assets, address receiver, address owner) public override returns (uint256 shares) {
```

Comment:
can we get rid of the `shares` return variable?

Reply:
Done